t/python/t0013-job-list-inactive.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python
+
+###############################################################
+# Copyright 2014 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+
+import os
+import errno
+import sys
+import json
+import unittest
+import datetime
+import time
+from glob import glob
+
+import yaml
+import six
+
+import flux
+from flux import job
+from flux.job import Jobspec, JobspecV1, ffi
+
+
+def __flux_size():
+    return 1
+
+
+class TestJob(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.fh = flux.Flux()
+
+    @classmethod
+    def submitJob(self):
+        compute_jobreq = JobspecV1.from_command(
+            command=["sleep", "0"], num_tasks=2, num_nodes=1, cores_per_task=1
+        )
+        compute_jobreq.cwd = os.getcwd()
+        compute_jobreq.environment = dict(os.environ)
+        flux.job.submit(self.fh, compute_jobreq, waitable=True)
+
+    attrs = [
+        "userid",
+        "state",
+        "name",
+        "ntasks",
+        "t_submit",
+        "t_run",
+        "t_inactive",
+    ]
+
+    # should return an empty list if there are no inactive jobs
+    def test_00_list_inactive_expect_empty_list(self):
+        rpc_handle = flux.job.job_list_inactive(
+            self.fh, time.time() - 3600, 10, self.attrs
+        )
+
+        try:
+            jobs = rpc_handle.get_jobs()
+        except EnvironmentError as e:
+            print("{}: {}".format("rpc", e.strerror), file=sys.stderr)
+            sys.exit(1)
+
+        self.assertEqual(len(jobs), 0)
+
+    # make sure one job is read from RPC
+    def test_01_list_inactive_success(self):
+        # submit a sleep 0 job
+        self.submitJob()
+
+        time.sleep(1)
```

Comment:
If possible, avoid sleeping in tests because this will slow the testsuite down. It appears that the job was submitted with `waitable=True`, so try using `job.wait()` here to wait just until the job completes to move on to the `job_list_inactive()` call.